modules/drivers/mcp230/mcp230.js

```diff
@@ -32,138 +32,104 @@ const PULLUPS_8  = 0b00000000;
 const INPUTS_16  = 0b1111111111111111;
 const PULLUPS_16 = 0b0000000000000000;
 
-// Default Config (Both)
-const ADDRESS = 0x20;
-const HZ = 0x00;
-const SDA = 0x04;
-const SCL = 0x05;
-
 // Modes (Ref: modules/pins/digital)
 const INPUT = 0;
 const INPUT_PULLUP = 1;
 const OUTPUT = 8;
 
 class Expander extends SMBus {
-  constructor(dictionary) {
+  constructor(dictionary = { address: 0x20 }) {
+    super(dictionary);
 
-    const offset = (dictionary.pins >> 3) - 1;
-    const {
-      address = ADDRESS,
-      hz = HZ,
-      sda = SDA,
-      scl = SCL,
+    this.offset = (this.length >> 3) - 1;
 
+    const {
       // User specific state initialization settings
-      inputs = offset ? INPUTS_16 : INPUTS_8,
-      pullups = offset ? PULLUPS_16 : PULLUPS_8,
-      pins,
-      reg,
-      reg: { IODIR, GPIO, GPPU }
-    } = dictionary || {};
-
-    super({ address, hz, sda, scl });
-
-    // trace(`IODIR: 0x${IODIR.toString(16)}\n`);
-    // trace(`GPIO: 0x${GPIO.toString(16)}\n`);
-    // trace(`GPPU: 0x${GPPU.toString(16)}\n`);
+      inputs = this.offset ? INPUTS_16 : INPUTS_8,
+      pullups = this.offset ? PULLUPS_16 : PULLUPS_8,
+    } = dictionary;
```

Comment:
Removals/Updates per **I2C** section of https://github.com/Moddable-OpenSource/moddable/pull/21#issuecomment-353265714